src/server/httpServer.js

```diff
@@ -4,15 +4,15 @@ const next = require('next');
 const open = require('opn');
 
 const DEV = process.env.NODE_ENV !== 'production';
-const SERVER_ADRESS = 'http://localhost:3000';
 
-const serverHandler = (err) => {
-  if (err) throw err;
-  open(SERVER_ADRESS);
-  console.log(`> Ready on ${SERVER_ADRESS}`);
-};
+const startServer = async (port) => {
```

Comment:
I must have a problem with my eslint because I had rule that prevents me from making this change
Eslint Error : Expected parentheses around arrow function argument having a body with curly braces. (arrow-parens)

Reply:
My bad, I'm misleading this codestyle with another one 😓